core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -254,8 +254,16 @@ object SparkEnv extends Logging {
           if (port == 0 || rpcEnv.address == null) {
             port
           } else {
-            rpcEnv.address.port + 1
+
+            if (Utils.hasPortRange(conf)){
```

Comment:
Oh, yes, I couldn't enable the rule to enforce spaces before `{` because of false positives inside interpolated strings (something like `s"blah ${foo}"` would trigger erroneously).